jujugui/static/gui/src/test/test_browser_charm_details.js

```diff
@@ -330,9 +330,11 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
     it('_addCharmEnvironment displays the config panel', function(done) {
-      var fakeStore = new Y.juju.charmstore.APIv4({
-        charmstoreURL: 'localhost/'
-      });
+      window.juju_config = {
```

Comment:
That should work for 99.9% of the cases. I'm a little concerned however that we may set a global `window.juju_config` and then just need to modify it in the tests so you should probably store the 'old' value and then reset it it back to that, even if now that's `undefined`